subtests/docker_cli/cp/cp.py

```diff
@@ -207,3 +208,72 @@ def verify_files_number(self, copied_number, expected_number):
         self.loginfo("Success, copied %d files from container, "
                      "expected number from configuration %d"
                      % (copied_number, expected_number))
+
+
+class volume_mount(CpBase):
+    """
+    Regression between docker-1.10.3 and 1.12.
+
+    See: https://github.com/docker/docker/issues/27773
+    """
+
+    def initialize(self):
+        super(volume_mount, self).initialize()
+
+        vol = 'v_' + utils.generate_random_string(8)
+        subargs = ['create', '--name', vol]
+        mustpass(DockerCmd(self, 'volume', subargs).execute())
```

Comment:
Sure, but then validating the name is acceptable and doesn't clash becomes a pain.  By the time anyone investigates, all the other volumes are gone, so PD is impossible.  Maybe a "cheap" compromise could be to just ``logdebug`` the output from ``docker volume ls``?